perma_web/perma/templates/base-responsive.html

```diff
@@ -14,16 +14,16 @@
     <!-- -->
 
     <meta name="twitter:card" content="summary">
-    <meta name="twitter:site" content="@HarvardLIL">
-    <meta name="twitter:title" content="Websites change. Perma Links don’t.">
+    <meta name="twitter:site" content="@permacc">
+    <meta name="twitter:title" content="Websites change. Perma Links don't.">
     <meta name="twitter:description" content="Perma.cc helps scholars, journals, courts, and others create permanent records of the web sources they cite.">
-    <meta name="twitter:image" content="{{ STATIC_URL }}img/create-step-4.png">
+    <meta name="twitter:image" content="{{ request.get_host }}{{ STATIC_URL }}img/create-step-4.png">
```

Comment:
keeping the non-logo sharing image here because of twitter's sharing rules: 
"You should not use a generic image such as your website logo, author photo, or other image that spans multiple pages"
https://dev.twitter.com/cards/types/summary-large-image